Applications/Spire/Source/Login/LoginWindow.cpp

```diff
@@ -4,19 +4,35 @@
 #include <QVBoxLayout>
 #include "Spire/Login/ChromaHashWidget.hpp"
 #include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Ui/Box.hpp"
```

Comment:
What is ```Box.hpp``` used for?

Reply:
The Box header defines the basic style properties, but neither the Button nor the Icon header include them.